### ISSUE TITLE
Fix WCAG 1.3.5: Identify Input Purpose issues

### ISSUE THREAD

**Issue by juliatufts** [scottrigby/basekamp-next#5]
Fix the following issues related to WCAG 1.3.5: Identify Input Purpose, see also: [full accessibility report](https://github.com/basekamp/basekamp-next/wiki/Accessibility-Evaluation-Report-2026).

## Contact Page

Name and email fields should have autocomplete
